tests/sequence_break/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2,12 +2,75 @@
 
 exports[`break.js 1`] = `
 const f = (argument1, argument2, argument3) =>
-  (doSomethingWithArgument(argument1), doSomethingWithArgument(argument2),argument1)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  (doSomethingWithArgument(argument1), doSomethingWithArgument(argument2),argument1);
+(function(){
+	return aLongIdentifierName, aLongIdentifierName, aLongIdentifierName, aLongIdentifierName;
+});
+aLongIdentifierName, aLongIdentifierName, aLongIdentifierName, aLongIdentifierName;
+a.then(() => (aLongIdentifierName, aLongIdentifierName, aLongIdentifierName, aLongIdentifierName));
+for (aLongIdentifierName = 0, aLongIdentifierName = 0, aLongIdentifierName = 0, aLongIdentifierName = 0; test; update) {}
+(a = b ? c : function() { return 0; }),
+  (a = b ? c : function() { return 0; }),
+  (a = b ? c : function() { return 0; }),
+  (a = b ? c : function() { return 0; }),
+  (a = b ? c : function() { return 0; });~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 const f = (argument1, argument2, argument3) => (
   doSomethingWithArgument(argument1),
   doSomethingWithArgument(argument2),
   argument1
 );
+(function() {
+  return (
+    aLongIdentifierName,
+    aLongIdentifierName,
+    aLongIdentifierName,
+    aLongIdentifierName
+  );
+});
+aLongIdentifierName,
+  aLongIdentifierName,
+  aLongIdentifierName,
+  aLongIdentifierName;
+a.then(
+  () => (
+    aLongIdentifierName,
+    aLongIdentifierName,
+    aLongIdentifierName,
+    aLongIdentifierName
+  )
+);
+for (
+  aLongIdentifierName = 0,
+    aLongIdentifierName = 0,
+    aLongIdentifierName = 0,
+    aLongIdentifierName = 0;
```

Comment:
We just didn't break at all previously - you can see in the first previous commit [what things looked like](https://github.com/prettier/prettier/pull/2388/commits/b18b6846f30c89cb93b985008e7eba7ff1216953#diff-e1baa6054f27ebf4c3ea72fb5aea5bc5R35) before the change.

And yup, we [indent declarations](https://prettier.io/playground/#%7B%22content%22%3A%22for%20(var%20aLongIdentifierName%20%3D%200%2C%20aLongIdentifierName%20%3D%200%2C%20aLongIdentifierName%20%3D%200%2C%20aLongIdentifierName%20%3D%200%3B%20test%3B%20update)%20%7B%7D%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%2C%22ast%22%3Afalse%7D%7D) here.